OleksandraKotovska/HW4/1.py

```diff
@@ -0,0 +1,10 @@
+temperature_celsius = float(input("Enter the temperature in Celsius: "))
```

Comment:
good job

look at this version of code

temperature_celsius = float(input("Enter the temperature in Celsius: "))
absolute_zero_celsius = -273.15
max_valid_celsius = 1000000

if absolute_zero_celsius <= temperature_celsius <= max_valid_celsius:
    fahrenheit = (temperature_celsius * 9/5) + 32
    print(f"{temperature_celsius:.2f}\u00b0C is equivalent to {fahrenheit:.2f}\u00b0F")
elif temperature_celsius < absolute_zero_celsius:
    print(f"Error: Temperature below absolute zero ({absolute_zero_celsius:.2f}\u00b0C)")
else:
    print(f"Error: Temperature above {max_valid_celsius}\u00b0C")